data/json/items/comestibles/med.json

```diff
@@ -561,6 +561,29 @@
         }
     },
     {
+        "id" : "disinfectant_powdered",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "powdered disinfectant",
+        "description" : "A powdered form of disinfectant. Could be used as a disinfectant, or mixed with water to become disinfectant",
```

Comment:
Double space after period in the first sentence. Period at the end of the second sentence. 
Also, maybe rephrase second sentence a bit? Like "Could be used as is, or mixed with water to become more potent version."?

Reply:
I did fix this in my personal version, I supposed I forgot to do this in the git files.